ansible/roles/openshift_udp_limits/files/iptables_udp.conf

```diff
@@ -0,0 +1,5 @@
+# Ansible-managed
+:msg, startswith, "UDP:" -/var/log/iptables.log
```

Comment:
Is there a way to get a more unique prefix via the iptables configuration?  Seems like lots of other logs might begin with `UDP:` as well.